application/app.py

```diff
@@ -398,6 +399,11 @@ def open_project(self):
         else:
             pass  # If no folder selected, don't load anything.
 
+    def open_feedback(self):
+        url = QtCore.QUrl('https://docs.google.com/forms/d/e/1FAIpQLSeTRwZlMUwNrbv9Nw-BddsOBGrCQjR5YXHbloPirRzB3-QoFA/viewform')
+        if not QtGui.QDesktopServices.openUrl(url):
+            QtGui.QMessageBox.warning(self, 'Connecting to Feedback', 'Could not open feedback form')
```

Comment:
I've not been able to test this Message Box warning.  Rather, if my internet is shut off, the url still opens in the browser, but the browser indicates an error message.  How were you testing this?